test/unit/repository/dnf_test.py

```diff
@@ -163,9 +163,14 @@ def test_add_repo_inside_buildservice(
             )
 
     @patch('kiwi.repository.dnf.RpmDataBase')
-    def test_setup_package_database_configuration(self, mock_RpmDataBase):
+    @patch('kiwi.command.Command.run')
+    @patch('kiwi.repository.zypper.Path.create')
```

Comment:
Why are we pulling in the Zypper module here?

Reply:
ouch my bad (copy & paste leftover), wondering how it can be that the test passes :thinking:

Reply:
Luck? There are two instances of this I see in this diff, so you probably want to fix that and make sure the tests pass still...

Reply:
fixed